ocs_ci/ocs/platform_nodes.py

```diff
@@ -2574,6 +2575,21 @@ def create_and_attach_nodes_to_cluster(self, node_conf, node_type, num_nodes):
         self.create_nodes(node_conf, node_type, num_nodes)
 
 
+
+class ROSANodes(AWSNodes):
+    """
+    ROSA EC2 instances class
+
+    """
+    def __init__(self):
```

Comment:
Why is `__init__` function overridden? It looks identical to the function in AWSNodes.